apps/aqua/src/auth/router.ts

```diff
@@ -41,13 +42,82 @@ export async function callback(c: TealContext) {
       maxAge: 60 * 60 * 24 * 365,
     });
 
+    if(params.get("spa")) {
+      return c.json({
+        provider: "atproto",
+        jwt: did,
+        accessToken: did,
+      })
+    }
+
     return c.redirect("/");
   } catch (e) {
     console.error(e);
     return Response.json({ error: "Could not authorize user" });
   }
 }
 
+
+// Refresh an access token from a refresh token. Should be only used in SPAs.
+// Pass in 'key' and 'refresh_token' query params.
+export async function refresh(c: TealContext) {
```

Comment:
oops forgot to mount this on the router 🙃 should be fine for now though, we aren't using an SPA